app/components/publications/publication/proofs/proof-request-modal.js

```diff
@@ -125,10 +125,11 @@ export default class PublicationsPublicationProofsProofRequestModalComponent ext
   *setEmailFields() {
     const publicationFlow = this.args.publicationFlow;
     const identification = yield publicationFlow.identification;
-
+    const urgencyLevel = yield publicationFlow.urgencyLevel;
     const mailParams = {
       identifier: identification.idName,
       shortTitle: publicationFlow.shortTitle,
+      urgent: urgencyLevel?.isUrgent,
```

Comment:
Param ook `isUrgent` noemen zodat het duidelijk is dat het een boolean is (en in lijn met`urgencyLevel.isUrgent`)?